src/reducers/ticketReducers.js

```diff
@@ -45,6 +46,12 @@ const addIncidentsToState = (state, incidents) => {
   return newState
 }
 
+const removeTicketFromState = (state, ticketId) => {
+  let newState = { ...state }
+  let updatedState = _.omit(newState, ticketId)
```

Comment:
No need to add a new dependency just for this one method. This ES6 method should do the trick:
```
let newState = { ...state }
Reflect.deleteProperty(newState, ticketId)
return newState
```

Reply:
This should work too:
```
const { [ticketId]: toRemove, ...newState } = state
return newState
```

Reply:
I'd looked at that one before and it seems like a good move.  Thanks!